covalent_dispatcher/_cli/service.py

```diff
@@ -569,3 +569,9 @@ def cluster(
         loop.run_until_complete(_cluster_scale(admin_server_addr, nworkers=scale))
         click.echo(f"Cluster scaled to have {scale} workers")
         return
+
+
+@click.command()
+def config() -> None:
```

Comment:
Could you add a docstring?. Otherwise `covalent --help` shows no description for the `config` command`.